package.json

```diff
@@ -20,6 +20,9 @@
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
     "test:e2e": "jest --config ./test/jest-e2e.json"
   },
+  "eslintConfig": {
```

Comment:
`scripts` still contains reference to `prettier --fix` use `eslint --fix` instead

Reply:
@xenide is looking into it on another PR